metaplugins/blockers.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+"""A generalized framowork for handling test blockers.
+
+Currently handling Bugzilla nad GitHub issues. For extensions, see this file and
+:py:mod:`utils.blockers`.
+
+If you want to mark test with blockers, use meta mark ``blockers`` and specify a list of the
+blockers. The blockers can be directly the objects of :py:class:`utils.blockers.Blocker` subclasses,
+but you can use just plain strings that will get resolved into the objects when required.
+
+Example comes:
+
+.. code-block:: python
+
+    @pytest.mark.meta(
+        blockers=[
+            BZ(123456),             # Will get resolved to BZ obviously
+            GH(1234),               # Will get resolved to GH if you have default repo set
+            GH("owner/repo:issue"), # Otherwise you need to use this syntax
+            123456,                 # Will resolve to BZ
+            "BZ#123456",            # Will resolve to BZ too
+            "Issue #123",           # Will resolve to GH (you need default repo set for this)
```

Comment:
I'm thinking it'd be good to simplify around a limited set, and probably always identify what tracker to use and use string literals.

e.g. `"BZ#123456"`, `"GH#123"`, `"GH#owner/repo:issue"`. The might(?) simplify things below a little bit, since it allows for getting rid of the regular expressions (`tracker, bugid = blocker.split("#")`)